### ISSUE TITLE
possible dependence cycle in meta code

### ISSUE THREAD

**Issue by martingabelmann** [FlexibleSUSY/FlexibleSUSY#567]
Hi!

I've been playing with FS and got a error message in a rather early stage when building a model (no matter which model), following all instructions from the readme:
```
>> make
Building models/lowNMSSM/00_DELETE_ME_TO_RERUN_METACODE
Mathematica 11.1.0 Kernel for Linux x86 (64-bit)
Copyright 1988-2017 Wolfram Research, Inc.

In[1]:= SARAH 4.14.5
by Florian Staub, Mark Goodsell and Werner Porod, 2020
contributions by M. Gabelmann, K. Nickel

References:
  Comput.Phys.Commun.181 (2010) 1077-1086. (arXiv:0909.2863)
  Comput.Phys.Commun.182 (2011) 808-833. (arXiv:1002.0840)
  Comput.Phys.Commun.184 (2013) 1792-1809. (arXiv:1207.0906)
  Comput.Phys.Commun.185 (2014) 1773-1790. (arXiv:1309.7223)
  Eur.Phys.J.C 74 (2014) 8, 2992 (arXiv:1405.1434)
  Eur.Phys.J.C 75 (2015) 1, 32 (arXiv:1411.0675)
  Eur.Phys.J.C 75 (2015) 6, 290 (arXiv:1503.03098)
  Eur.Phys.J.C 77 (2017) 11, 758 (arXiv:1703.09237
  Eur.Phys.J.C 77 (2017) 11, 757 (arXiv:1706.05372)
  Eur.Phys.J.C 78 (2018) 8, 649 (arXiv:1805.07306)
Download and Documentation:
  http://sarah.hepforge.org

Start evaluation of a model with:
   Start["Name of Model"]
e.g. Start["MSSM"] or Start["NMSSM","CKM"]
To get a list with all installed models, use ShowModels

Needs::nocont: Context Parameters` was not created when Needs was evaluated.

Needs::nocont: Context Parameters` was not created when Needs was evaluated.

Needs::nocont: Context Parameters` was not created when Needs was evaluated.

General::stop: Further output of Needs::nocont
     will be suppressed during this calculation.
```
followed by a lot of other errors related to the fact that the `Parameters` context was not loaded...

I used a freshly installed version of sarah for this test (appended to `$Path` as explained in the readme) and only tried the built-in models (e.g. HSSUSY or lowNMSSM). No errors/missing libraries during the createmodel/configure steps...

Im not familiar with the code but I found a solution that works for me:
I could trace it back to the `models/<modelname>/start.m`.  If I prepend the line
```
Needs["Parameters`",  FileNameJoin[{Directory[], "meta", "Parameters.m"}]]
```
to the already existing line
```
Needs["FlexibleSUSY`",  FileNameJoin[{Directory[], "meta", "FlexibleSUSY.m"}]];
```
and rerun make, the build succeeds. Also the generated spectrum generator seems to run fine.

Cheers,
Martin

**Comment by martingabelmann** [comment]
Hi Alex,
thanks for the quick answer! I tried several Mathematica versions (including 13.2 and 13.0) in the meantime, without success. However, I also tried it on my laptop rather than the office computers and there it runs fine out of the box (on my laptop I only have 13.0.1 available while at the institute I tried from 11 up to 13).
I even removed everything (except sarah) from the `init.m` and the `$Path` variable to make sure nothing else is interfering here, w/o any noticeable change...

Since I cannot reproduce it on any other available installation and since I do have a working solution for the institute installation, its not really an issue..
Thanks!
